cmd/build.go

```diff
@@ -52,23 +52,23 @@ func newBuildCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// buildCmd provides the ability run local builds, or setup/delete Tekton builds, for an appsody project
 	var buildCmd = &cobra.Command{
 		Use:   "build",
-		Short: "Locally build a container image of your Appsody project",
-		Long: `Build a local container image of your Appsody project. The stack along with your Appsody project is first extracted to a local directory, before the docker build is run.
+		Short: "Build a local container image of your Appsody project.",
+		Long: `Build a local container image of your Appsody project. The stack along with your Appsody project, is extracted to a local directory before the docker build is run.
```

Comment:
Needs a comma after "stack": 
"The stack, along with your Appsody project, ..."